lib/evmone_precompiles/bls.cpp

```diff
@@ -365,13 +359,11 @@ void store(uint8_t _rx[128], const blst_fp2& _x) noexcept
     static constexpr auto PAIR_SIZE = P1_SIZE + P2_SIZE;
     assert(size % PAIR_SIZE == 0);
 
-    const auto npairs = size / PAIR_SIZE;
-    auto ptr = _pairs;
-    blst_fp12 acc = *blst_fp12_one();
-
     const auto Qlines = std::make_unique_for_overwrite<blst_fp6[]>(68);
```

Comment:
I would remove `Qlines` and run regular pairing without lines precomputation.

Reply:
Done in https://github.com/ethereum/evmone/pull/1155.